__init__.py

```
@@ -116,57 +157,37 @@ def handle_weather(self, message: Message):
+        elif self.voc_match(utt, "couple"):
             days = 2
-        elif self.voc_match(message.data["utterance"], "few"):
+        elif self.voc_match(utt, "few"):
             days = 3
         else:
-            days = int(extract_number(message.data["utterance"], lang=self.lang))
-        self._report_multi_day_forecast(message, days)
+            try:
+                days = int(extract_number(message.data["utterance"], lang=self.lang))
+            except:
+                pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)